TriggerRepoBuild/Program.cs

```diff
@@ -187,6 +192,7 @@ static void RecursiveDelete(string dir)
             bool deleted = false;
             foreach (var d in Directory.GetDirectories(dir))
             {
+                deleted = false;

```

Comment:
I recommend declaring `deleted` here and removing the declaration on line 192.  The variable isn't used outside the loop.

Reply:
Done